uk.co.compucorp.civicrm.hrcore/gulpfile.js

```diff
@@ -87,20 +113,22 @@ var Promise = require('es6-promise').Promise;
       return Promise.reject(new Error());
     }
 
-    return new Promise(function (resolve) {
-      gulp.src(BACKSTOP_DIR + FILES.tpl)
-      .pipe(file(destFile, tempFileContent()))
-      .pipe(gulp.dest(BACKSTOP_DIR))
-      .on('end', function () {
-        var promise = backstopjs(command, {
-          configPath: BACKSTOP_DIR + destFile,
-          filter: argv.filter
-        })
-        .catch(_.noop).then(function () { // equivalent to .finally()
-          gulp.src(BACKSTOP_DIR + destFile, { read: false }).pipe(clean());
-        });
-
-        resolve(promise);
+    return getRolesAndIDs().then(function (contactRolesAndIDsMap) {
```

Comment:
maybe `contactIdsByRoles` is more descriptive

Reply:
Done.